src/main/java/se/sundsvall/casedata/api/AppealResource.java

```diff
@@ -20,22 +25,23 @@
 import org.zalando.problem.Problem;
 import org.zalando.problem.violations.ConstraintViolationProblem;
 
+import se.sundsvall.casedata.api.model.AppealDTO;
+import se.sundsvall.casedata.api.model.PatchAppealDTO;
+import se.sundsvall.casedata.service.AppealService;
+import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
+
 import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
 import io.swagger.v3.oas.annotations.media.Content;
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import io.swagger.v3.oas.annotations.tags.Tag;
-import jakarta.validation.Valid;
-import se.sundsvall.casedata.api.model.AppealDTO;
-import se.sundsvall.casedata.api.model.PatchAppealDTO;
-import se.sundsvall.casedata.service.AppealService;
-import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
 
 @RestController
 @Validated
-@RequestMapping("/{municipalityId}/appeals")
```

Comment:
It is indeed strange to add something new and create a PR without any tests for it.